src/libs/Navigation/Navigation.ts

```diff
@@ -173,12 +172,9 @@ function goBack(fallbackRoute, shouldEnforceFallback = false, shouldPopToTop = f
 
 /**
  * Update route params for the specified route.
- *
- * @param {Object} params
- * @param {String} routeKey
  */
-function setParams(params, routeKey) {
-    navigationRef.current.dispatch({
+function setParams(params: Record<string, unknown>, routeKey: string) {
+    navigationRef.current?.dispatch({
```

Comment:
Do you know why we need `current` to call `dispatch` but can call e.g. `getRootState` without it?

Reply:
I'll investigate it, @adamgrzybowski maybe you have some idea how it works?

Reply:
I discussed it with Adam and decided to leave it as is for now and investigate in the future.